humanlayer-wui/src/lib/daemon/http-client.ts

```diff
@@ -123,6 +123,17 @@ export class HTTPDaemonClient implements IDaemonClient {
 
   // Session Management Methods
 
+  async getSlashCommands(params: {
+    sessionId: string
+    query?: string
+  }): Promise<{ data: Array<{ name: string; source: 'local' | 'global' }> }> {
+    await this.ensureConnected()
+
+    const response = await this.client!.getSlashCommands(params)
+
+    return response as { data: Array<{ name: string; source: 'local' | 'global' }> }
```

Comment:
minor, but a thing to watch out for, there should be return type available for this